core/src/main/resources/defaults/devblueprints/7.2.15/cdp-data-engineering.bp

```diff
@@ -0,0 +1,124 @@
+{
+   "description":"[DEV] 7.2.15 - Data Engineering",
+   "blueprint":{
+      "cdhVersion":"7.2.15",
```

Comment:
it can be lower? Easier to test the upgrade if the version is 7.2.12 or stuff like this (less cluster provision, more upgrade on the same cluster)

Reply:
this cheap template is for testing high node count scale, not for upgrade, but if someone wants to test upgrade with it, they can extend it, but CM functionality is not validated, we plan to test only salt performance

Reply:
also upgrade should be tested with production template since it affects CM functionality heavily (runtime upgrade, services upgrade, etc.)

Reply:
it was just a hint for making easier our life, but if it cannot help to upgrade the test just the salt performance, ok for me